### PR TITLE
6308 mdbc integration

### PR DESCRIPTION
**Pull request by h0st**
Single mariadb enterprise and community recipe and one for maxscale

### CODE REVIEW

File: recipes/mdbrepos.rb [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
   end
 when "suse"
-  # Add the repo
-  template "/etc/zypp/repos.d/mariadb.repo.template" do
+  template "/etc/zypp/repos.d/mariadb.repo" do
     source "mariadb.suse.erb"
     action :create
   end
-  release_name = "test -f /etc/os-release && cat /etc/os-release | grep '^ID=' | sed s/'^ID='//g | sed s/'\"'//g || if cat /etc/SuSE-release | grep Enterprise &>/dev/null; then echo sles; else echo opensuse; fi"
+  release_name = "if cat /etc/SuSE-release | grep Enterprise &>/dev/null; then echo sles; else echo opensuse; fi"
```
Comment (ferustigris) [inline recipes/mdbrepos.rb]: Does it work in both sles 11&12?

File: recipes/mdbrepos.rb [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
   execute "Change suse on sles repository" do
-    command "cat /etc/zypp/repos.d/mariadb.repo.template | sed s/PLATFORM/$(" + release_name + ")/g > /etc/zypp/repos.d/mariadb.repo"
+  	command "cat /etc/zypp/repos.d/mariadb.repo | sed s/PLATFORM/$(" + release_name + ")/g > /etc/zypp/repos.d/mariadb.repo"
```
Comment (ferustigris) [inline recipes/mdbrepos.rb]: formatting

File: recipes/install_maxscale.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    path "/etc/selinux/config"
+    action :create
+  end
+end  # Turn off SElinux
+
+# Install packages
+case node[:platform_family]
+when "suse"
+  execute "install" do
+    command "zypper install maxscale"
```
Comment (ferustigris) [inline recipes/install_maxscale.rb]: Is it going to work?
1. Why is not just 'package  maxscale'?
2. Will zypper ask user to confirm installation?

File: recipes/mdbrepos.rb (hunk 1 above)
Comment (h0st) [inline recipes/mdbrepos.rb]: I test this code in aws sles 11/12 versions and it works. I can test it in sles boxes.

**Comment by ferustigris** [comment]
It looks good, but I have a little notes:
1. maxscale should be a separated coockbook (there are different products)
2. there are both MDBE and MDB should be able from the "box"
3. commented code should be removed

File: attributes/default.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
+#
+# MariaDB Enterprise/Community
```
Comment (ferustigris) [inline attributes/default.rb]: m.b. split initialization in different files (install_community, install_enterprise)?

File: recipes/install_maxscale.rb (hunk 3 above)
Comment (h0st) [inline recipes/install_maxscale.rb]: 1. for suse added repository and i install it via zypper same as for mariadb. package maxscale is 'else' branch. 
2. Yes, -y flag is missing.

File: attributes/default.rb (hunk 4 above)
Comment (h0st) [inline attributes/default.rb]: First cookbook version have different init files (install_community, install_enterprise), but after meeting I merged them.

Comment (ferustigris) [inline attributes/default.rb]: Why?

**Comment by ferustigris** [comment]
Can I ask you to forward pull-request to develop branch (I've created it, my bad)?

File: attributes/default.rb (hunk 4 above)
Comment (h0st) [inline attributes/default.rb]: I think, for further mariadb automation. I can clarify this question or some one else answer to it.

**Comment by h0st** [comment]
Yes, of course. Merged it?

**Comment by h0st** [comment]
Done. Forward to develop branch.

**Pull request closed without merging**